homeassistant/components/lutron_caseta.py

```diff
@@ -96,6 +98,21 @@ def name(self):
         """Return the name of the device."""
         return self._device_name
 
+    @property
+    def model(self):
+        """Return the model number of the device."""
+        return self._device_model
+
+    @property
+    def serial(self):
+        """Return the serial number of the device."""
+        return self._device_serial
+
+    @property
+    def unique_id(self):
+        """Return the unique ID of the device (model + serial)."""
+        return "{}.{}".format(self._device_model, self._device_serial)
```

Comment:
Wouldn't a serial be enough to provide a unique id?

Reply:
Probably. I don't know if it's the serial number for that model or the serial number for all devices they've produced. It's just an integer value.

Reply:
It must be the serial for the devices. Let's stick with just the serial.

Reply:
I'm worried about the super improbable case where somebody buys both dimmer switch 1234 and binary switch 1234, but probably serial is fine.